### ISSUE TITLE
Unable to navigate video

### ISSUE THREAD

**Issue by birdwing** [tubearchivist/tubearchivist#287]
I had to restart my server do to another issue.
When I came back to the video I was watching, it didn't leave off where I left it.

When I attempted to drag the position on the bar, it keeps snapping back to where it was at.
I cannot scroll ahead on the video.

Using the v0.2.0 update.

**Comment by birdwing** [comment]
It's worth noting, I can scroll _backwards_ but not forward.

**Comment by birdwing** [comment]
This appears to be intermittent. I cannot identify what causes it.

**Comment by bbilly1** [comment]
So the only thing that interferes with the timeline are the Sponsorblock timestamps. You could check if there is anything wrong there, like overlapping timestamps that will land you in a loop? See the segments with `/api/video/<video-id>/`.

**Comment by birdwing** [comment]
I did not have the Sponsorblock integration turned on at the time.
But I also have not seen this issue in a week or so, it may be related to some of the other issues people have been reporting.
I have also been getting stuttering, and video playback issues. Like the video freezing and "buffering" even though the progress bar shoes that the video is loaded.

So I will go ahead and close this issue because I believe what I saw will end up getting resolved as the other issues are fixed 👍
